ci-playmode-test.sh

```diff
@@ -1,17 +1,13 @@
-#!/usr/bash
+#!/usr/bin/env bash
 
 source ci-setup.sh
 
-${UNITY_EXECUTABLE:-xvfb-run --auto-servernum --server-args='-screen 0 640x480x24' /opt/Unity/Editor/Unity } \
+xvfb-run --auto-servernum --server-args='-screen 0 640x480x24' $UNITY_PATH/Editor/Unity \
         -batchmode \
         -logFile "$PROJECT_PATH/playmode-logs.txt" \
         -runTests \
         -testPlatform PlayMode \
         -testResults "$PROJECT_PATH/playmode-results.xml" \
-        -enableCodeCoverage \
```

Comment:
can we use left the coverage turned on?

Reply:
Can we add this later? I saw that the coverage output files are not deployed or copied anywhere, so it might make no sense right now.

Reply:
They are supposed to be uploaded to circle, using the store_artifacts https://circleci.com/docs/2.0/code-coverage/ if not, we can remove it. I thought that it would be nice to have that info available since the beginning 

https://github.com/decentraland/unity-renderer/blob/8a1a4ccd7dfdaa0d2f5d3deaf2da7a1c471c6d94/.circleci/config.yml#L113-L115

Reply:
@BrianAmadori